### DISCUSSION TITLE
Debugging Ideas

### DISCUSSION THREAD

**Discussion by EclecticGriffin** [cucapra/futil#593] [Ideas]
A collection place for the ideas about different debugging approaches/ideas.

**watchpoint debugging**
Log changes to a given a value and what drove the change (input -> output port)

**Comment by rachitnigam** [comment]
@EclecticGriffin just to wrap this discussion up–this is implemented now right?

**Reply by EclecticGriffin** [reply]
The "what drove the change" part doesn't really exist, but I think we can still close this discussion
